### PR TITLE
- developed UI for to-do list.

### PR DESCRIPTION
**Pull request by SudevSanthosh**
Developed UI design for to-do-list using HTML and CSS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet"href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <title>To-Do-List</title>
+</head>
+<body>
+    
+<div id="myDIV" class="header">
+  <h2 class="my-to-do-list">My To Do List</h2>
```
Comment (dheeraj-murali) [inline index.html]: Make class names generic and have it represented then use of element Eg: `title`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  
+  <input type="text" class="to-do-input" placeholder="Enter your task...">
+  <span  class="add-button"> <i class="fa fa-plus"></i>&ensp; Add Item</span>
```
Comment (dheeraj-murali) [inline index.html]: Clickable elements should always be a `<button />` or `<a />`

File: index.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    height: 30px;
+    width: 50%;
+    border-radius: 11px;
+  border:none;
+
+}
+.to-do-input:focus{
+    
+
+   }
```
Comment (dheeraj-murali) [inline index.css]: Shouldn't keep unused classes like this

File: index.css [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+}
+ul li:nth-child(odd):hover{
+    background-color: #84817a;
+    color: white;
+}
+ul li:nth-child(even):hover
+{
+    background-color: #84817a;
+    color: white;
+}
```
Comment (dheeraj-murali) [inline index.css]: It's always safe to have class names rather than styling an element

File: index.css [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+.input-icons {
+    width: 100%;
+    margin-bottom: 10px;
+    float: right;
+}
+  
+.icon {
+    padding: 10px;
+    min-width: 40px;
+}
```
Comment (dheeraj-murali) [inline index.css]: Are we using these?

**Review by dheeraj-murali — CHANGES_REQUESTED** [review]
UI is broken in iPhone SE
image


- I think you missed to add the task complete button
- also styles for completed tasks

File: index.html (hunk 2 above)
Comment (dheeraj-murali) [inline index.html]: This should be inside a form

**Review by dheeraj-murali — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (SudevSanthosh) [reply index.html]: Added button and placed the line 18 and 19 inside form.

File: index.css (hunk 3 above)
Comment (SudevSanthosh) [reply index.css]: removed

File: index.css (hunk 4 above)
Comment (SudevSanthosh) [reply index.css]: added class to those elements

File: index.css (hunk 5 above)
Comment (SudevSanthosh) [reply index.css]: removed unwanted classes

File: index.html (hunk 2 above)
Comment (SudevSanthosh) [reply index.html]: done

**Pull request merged**